### ISSUE TITLE
Convert the blog (currently not working) to a static page using Jekyll

### ISSUE THREAD

**Issue by prock-fife** [fifengine/fifeweb#4]
https://help.github.com/articles/using-jekyll-as-a-static-site-generator-with-github-pages/

**Comment by emilhem** [comment]
I think this is a great idea, I have some experience with Jekyll.
Is the page hosted on GitHub now? I don't see the CNAME file required.

**Comment by prock-fife** [comment]
The blog is not currently hosted on github but is hosted on a VM in the cloud along with our website.   It was a wordpress blog site but it got hacked and I had to shut it down because whatever they did broke it (the blog link on the fifengine.net website just points back to the homepage instead of the blog).  I'd like to export the posts from the database but if we start from scratch that is okay as well.  I would like to host the blog on github at a minimum but the entire site would be the end goal.

**Comment by emilhem** [comment]
If you extract the blog material I don't see any problem in trying to convert it into Jekyll.

**Comment by prock-fife** [comment]
okay I'll look at running an export.  I have the mysql dumps if that works for you.  There is a bunch of spam in the content that will need to be cleaned up.

**Comment by emilhem** [comment]
That works. Did you post anything after the spam?

**Comment by prock-fife** [comment]
The spam is embedded in the posts but won't be difficult to remove.

**Comment by emilhem** [comment]
Tarball it and compress it with xz and upload it somewhere. I can't imagine there being anything private in the dump except password hashes?

**Comment by prock-fife** [comment]
@emilhem are you still interested in doing this?  We now have a base jekyll site set up which is hosted at github.

**Comment by emilhem** [comment]
I can always give it a shot.

On a sidenote, the link to docs and forum are broken. Also make sure that you set up the place where fifengine.net is hosted to redirect to the www version (ie www.fifengine.net). This can sometimes be achived from the DNS provider. Otherwise if you have the possibility of hosting a tiny webserver that basically just redirects the user going to the non www version to the www version.

**Comment by prock-fife** [comment]
Yeah the forum is broken at the moment.  The host upgraded PHP and SMF doesn't like that.  We are working on a solution to that.  Also nice catch on the fifengine.net.  I'm going to put a small redirect in there to www.  The docs link however should be working.  I'll have to check that out.

**Comment by prock-fife** [comment]
Docs have been fixed.

**Comment by prock-fife** [comment]
fifengine.net now redirects to www.fifengine.net

**Comment by prock-fife** [comment]
Here is a little tutorial on how to get a local version of the site running locally.  https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/

I was able to do this on windows without any issues.

**Comment by prock-fife** [comment]
Not sure how exactly to get in contact with you.  Can you stop by IRC sometime and PM your email?  I'll send you the extracts.

**Comment by emilhem** [comment]
There's no information regarding the IRC to be found on the website, but I found it in the Fifengine repo.

I'm on the IRC now via WeeChat inside a Screen that is relayed to my phone. I will check every now and then.

**Comment by jakoch** [comment]
Closing, as this issue can not be solved without the missing data (which probably not available anywhere anymore).
